app/controllers/user_controller.rb

```diff
@@ -0,0 +1,44 @@
+require 'sinatra'
+require_relative '../models/user_model'
+require_relative '../services/aws_service'
+
+class UserController < Sinatra::Base
+  include AwsService
+
+  configure do
+    set :views, 'app/views'
+  end
+
+  before do
+    @csv_data = fetch_csv_data(ENV['AWS_S3_BUCKET'] || 'hd-data-2023', ENV.fetch('AWS_FILE_NAME', nil))
```

Comment:
[suggestion]
It would be probably better to load the latest file in the bucket instead, no?